src/main/java/io/github/wimdeblauwe/errorhandlingspringbootstarter/handler/GqlAccessDeniedExceptionHandler.java

```diff
@@ -0,0 +1,51 @@
+package io.github.wimdeblauwe.errorhandlingspringbootstarter.handler;
```

Comment:
Maybe we can put all GraphQL related handlers in a `graphql` sub-package?

Reply:
Would you want the Gql handlers to be under the root package or under root/handler sub-package?